src/Raven.Client/Documents/Operations/Replication/PullReplicationAsHub.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using Raven.Client.Documents.Replication;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Replication
+{
+    public sealed class PullReplicationAsHub : ExternalReplication
```

Comment:
in 5.4 this is `internal` should it be here as well?

Reply:
I saw that, will be handled during merge